feinstaub/feinstaub/urls.py

```diff
@@ -15,4 +22,4 @@
                            namespace='rest_framework')),
     url(r'^get-auth-token/', obtain_auth_token),
     url(r'^adddevice/', AddSensordeviceView.as_view(), name='adddevice'),
-]
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
What does this do?

Reply:
@DavidLemayian assets were being served by nginx https://github.com/CodeForAfricaLabs/sensors.AFRICA-AQ-api/blob/master/nginx.conf#L32 which moved to to sigil. With volumes gone we are serving locally. I think we can use http://whitenoise.evans.io/en/stable/django.html for that.